testing/mod.ts

```diff
@@ -193,15 +196,17 @@ async function runTestsSerial(
 /** Defines options for controlling execution details of a test suite. */
 export interface RunOptions {
   parallel?: boolean;
+  exitOnFail?: boolean;
 }
 
 /**
  * Runs specified test cases.
  * Parallel execution can be enabled via the boolean option; default: serial.
  */
-export async function runTests({ parallel = false }: RunOptions = {}): Promise<
-  void
-> {
+export async function runTests({
+  parallel = false,
+  exitOnFail = true
```

Comment:
Let's turn it off by default. This is just left over from early debugging. Now that we're more stable it makes sense.